app/builders/subscriber_auth_email_builder.rb

```diff
@@ -25,10 +25,39 @@ def call
   attr_reader :subscriber, :destination, :token
 
   def subject
-    "Confirm your email address"
+    return manage_your_subscriptions_subject if destination == "/manage/authenticate"
```

Comment:
I'm not sure this the destination https://github.com/alphagov/email-alert-frontend/blob/19223e768f85627fa917f456033fbcfc1b44b5fa/app/controllers/subscriber_authentication_controller.rb#L17